Mathlib/NumberTheory/ModularForms/QExpansion.lean

```diff
@@ -104,6 +93,60 @@ lemma analyticAt_cuspFunction_zero [NeZero n] [ModularFormClass F Γ(n) k] :
     (fun q hq ↦ (differentiableAt_cuspFunction _ _ hq).differentiableWithinAt)
     (by simpa only [ball_zero_eq] using Metric.ball_mem_nhds (0 : ℂ) zero_lt_one)
 
+/-- The `q`-expansion of a level `n` modular form, bundled as a `PowerSeries`. -/
+def qExpansion : PowerSeries ℂ :=
```

Comment:
Can you add this to "main definitions" in the doc at the beginning of the file? Thanks!